src/pyOpenMS/tests/unittests/test_mschromatogram_struct_view.py

```
@@ -0,0 +1,60 @@
+    chrom.set_peaks([[1.0, 2.0], [10.0, 20.0]])
+
+    arr = chrom.get_peaks_struct()
+
+    # Value correctness
+    assert np.isclose(arr[0, 0], 1.0)   # RT
+    assert np.isclose(arr[1, 1], 20.0)  # intensity
+
+    # PROVE ZERO-COPY
+    arr[0, 1] = 999.0
+
+    rt_copy, int_copy = chrom.get_peaks()
```

Ruff (0.15.2)

[warning] Unpacked variable `rt_copy` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)